### ISSUE TITLE
Footnote links and their backlinks are too small

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk_publishing_components#3440]
[Footnotes within the Govspeak component](https://components.publishing.service.gov.uk/component-guide/govspeak#footnotes) currently don't fail [2.5.8 Target Size (Minimum)](https://w3c.github.io/wcag/understanding/target-size-minimum.html). But that is only because there is an exception for interactive elements within inline text.

Note: 2.5.8 is part of the new draft WCAG 2.2 criteria. The latest standard is WCAG 2.1. WCAG 2.2 isn't published yet, so this is potentially subject to change.

But the small size of footnotes is still an accessibility issue, they are very difficult to click on with a mouse or finger on a touch device, especially for people with motor impairments. That’s worse when footnotes or backlinks are close to another link or footnote (which can also look different in other viewports).

When testing the smallest footnote link was only 5.62 x 18.5px and the backlink was 19 x 23.5px.
There is a worse example of footnotes under [Specialist content](https://components.publishing.service.gov.uk/component-guide/govspeak#specialist_content).

Screenshots of footnotes and backlinks with their dimensions:

<img width="395" alt="Screenshot of footnote" src="https://github.com/alphagov/govuk_publishing_components/assets/108893/7b7b5b49-07f6-4dc5-b157-577997ce6a71">
<img width="499" alt="Screenshot of backlink" src="https://github.com/alphagov/govuk_publishing_components/assets/108893/01267aa6-f117-4583-b1f7-f91922190c1d">

This was fixed in some areas on www.gov.uk already (e.g. partly [in this Govspeak PR](https://github.com/alphagov/govspeak/pull/192)) by changing:
* ‘1’ to ‘[footnote 1]’ (although a shorter '[note 1]' might also work)
* ‘↩’ to ‘Go to where this is referenced’ (although that was later changed to be in an aria-label and not visible text, there would need to be a bit more text or spacing to make the link wider)

That also shows that there is an issue of footnotes currently being inconsistent across www.gov.uk.

**Comment by andysellick** [comment]
Unfortunately probably not.

**Comment by nnagewad** [comment]
@selfthinker This is an interesting. Would you happen to have any live examples of this issue on-hand?

**Comment by andysellick** [comment]
@MartinJJones this issue is now closed - is that right?

**Comment by jon-kirwan** [comment]
We’ve reviewed the footnote and backlink target sizes against the WCAG 2.2 2.5.8 (Target Size – Minimum). As noted, these elements currently meet the requirement due to the inline text exception. That said, we agree that the small target size isn’t ideal particularly when links are positioned closely together.

As an incremental improvement, we'll create a backlog card to increase the line height of the "Go to where this is referenced" arrow by 0.5px (to 24px) to make it slightly easier to click or tap. Adjusting the footnote links themselves for a larger target area is more complex given the inline layout, and changing these within the text might cause other issues for readability and layout.
